ceremony/2022-09-27/repository/4.root.json

```diff
@@ -0,0 +1,144 @@
+{
```

Comment:
To confirm, there is not supposed to be 5.root uninitialized yet?

Reply:
correct, that occurs when publish / commit happens

Reply:
i was looking under the wrong folder!